setup.py

```diff
@@ -0,0 +1,31 @@
+from setuptools import setup
+import os.path
+import sys
+
+setup(
+    name = 'sphinx_fossasia_theme',
+    version = '0.0.1',
+    author = 'Ujjwal Bhardwaj',
+    author_email = 'ujjwalb1996@gmail.com',
+    url = 'https://github.com/fossasia/sphinx_fossasia_theme',
+    license = 'GNU',
+    description = 'A Sphinx theme specific to FOSSASIA\'s Projects',
+    packages = ['sphinx_fossasia_theme'],
+    include_package_data = True,
+    entry_points = {
+        'sphinx.html_themes': [
```

Comment:
Checking out an earlier version of sphinx documentation, it seems this might not be such a simple case. http://www.sphinx-doc.org/en/1.5.5/theming.html#using-a-theme. For now ignore this. Let's do that in a seperate PR